ophyd/areadetector/trigger_mixins.py

```diff
@@ -15,6 +15,7 @@ class MyDetector(PerkinElmerDetector, SingleTrigger):
 
 from ..ophydobj import DeviceStatus
 from ..device import BlueskyInterface
+from ..utils import set_and_wait
```

Comment:
Good catch. This missing import occurs in one of two lines that my test script did not touch.